packages/backend/src/modules/task/task.repository.ts

```diff
@@ -0,0 +1,41 @@
+import { CreateTaskCommand } from 'shared'
+import TaskModel from './task.model'
+import BoardModel from '../board/Board'
+
+class TaskRepository {
+  private _taskModel = TaskModel
+
+  private _boardModel = BoardModel
```

Comment:
It's TaskRepository so why you use BoardModel here? Create BoardRepository and call it from BoardService

Reply:
Done